packages/serverless-components/nextjs-cdk-construct/__tests__/fixtures/next-boilerplate/default-lambda/manifest.json

```diff
@@ -0,0 +1 @@
+{"buildId":"IOjzqKr7ylEZVL7Xy7FMd","logLambdaExecutionTimes":true,"pages":{"ssr":{"dynamic":{"/users/:id":{"file":"pages/users/[id].js","regex":"^\\/users(?:\\/([^\\/#\\?]+?))[\\/#\\?]?$"}},"nonDynamic":{"/users":"pages/users.js","/_error":"pages/_error.js"}},"html":{"dynamic":{},"nonDynamic":{"/about":"pages/about.html","/":"pages/index.html","/404":"pages/404.html"}}},"publicFiles":{"/planes.jpg":"planes.jpg"},"trailingSlash":false,"domainRedirects":{},"enableHTTPCompression":false}
```

Comment:
Format this JSON file for better readability (since they are used in unit tests)